lockfile.json

```diff
@@ -246,7 +249,7 @@
     "revision": "111b07762e86efab9a918b7c721f720c37e76b0a"
   },
   "verilog": {
-    "revision": "8f6b1f357d1231c420404b5f7a368a73c25adfa2"
+    "revision": "6fae7414fa854b5052bee9111b200e9137797f3d"
```

Comment:
please do not update verilog! It's incompatible.

Reply:
I should have know - did this the last time.
I try to put my glasses on next time ;)
Should be fixed